Makefile

```diff
@@ -8,7 +8,7 @@ INSTALL_STAMP = $(VENV)/.install.stamp
 TEMPDIR := $(shell mktemp -d)
 
 all: install
-install: $(INSTALL_STAMP)
+install: virtualenv $(INSTALL_STAMP)
```

Comment:
it will make the "install" target dependent on the "virtualenv" target, obliging `make` to run it before running "install". The target this also fixes the target "serve" since it depends on the target "install" ;)